pentaho/impl/src/main/java/pt/webdetails/cdf/dd/api/SyncronizerApi.java

```diff
@@ -287,6 +290,15 @@ public String saveDashboard( @FormDataParam( MethodParams.FILE ) @DefaultValue(
                                @FormDataParam( MethodParams.OPERATION ) String operation,
                                @Context HttpServletResponse response ) throws Exception {
 
+    response.setContentType( APPLICATION_JSON );
+    response.setCharacterEncoding( CharsetHelper.getEncoding() );
+
+    if( !isAllowSaveReport() ) {
+      String msg = "You have disabled this feature on settings.xml. Please set allow-save-report property to true.";
```

Comment:
Ideally, this would be on a i18n properties file.

Reply:
I did not find any resources bundle estruture in this project. I will leave this refactor for an improvement.